src/main/java/org/assertj/core/internal/StandardComparisonStrategy.java

```diff
@@ -82,6 +82,7 @@ public String asText() {
    */
   @Override
   public boolean areEqual(Object actual, Object other) {
+    System.out.println(actual.equals(other) + " tite");
```

Comment:
Please don't mind this and don't bother googling what it means either and if you do, I apologize.